glib/src/subclass/types.rs

```diff
@@ -86,6 +86,18 @@ pub unsafe trait InstanceStruct: Sized + 'static {
     }
 }
 
+/// Trait implemented by any type implementing `ObjectSubclassIs` to return the implementation, private Rust struct.
+pub trait ObjectSubclassIsExt: ObjectSubclassIs {
+    fn impl_(&self) -> &Self::Subclass;
```

Comment:
How about `imp` instead of `impl_`?

Reply:
Since this method will probably be used a lot when subclassing, I also think `imp` is better.
Now I've called it `impl_` because it does a similar thing to `InstanceStruct::impl_`.

Reply:
If you want to rename it, rename it everywhere :) But `imp` is also a word in English, so not the best choice :imp:

Reply:
Nevermind, let's keep `impl_`